modules/datadog/Makefile

```diff
@@ -19,14 +20,46 @@ define datadog_notify
 	@envsubst < $(MAKEFILE_DIR)/modules/datadog/$(1).json | $(CURL)
 endef
 
-## Notify datadog a deploy is starting
+## Notify datadog of a deploy is starting
 datadog\:notify-deploy-starting:
-	$(call datadog_notify,$(subst datadog:notify-,,$@))
+	$(call datadog_notify_deploy,$(subst datadog:notify-,,$@))
 
-## datadog datadog a deploy was successful
+## Notify datadog of a deploy was successful
 datadog\:notify-deploy-success:
-	$(call datadog_notify,$(subst datadog:notify-,,$@))
+	$(call datadog_notify_deploy,$(subst datadog:notify-,,$@))
 
-## datadog datadog a deploy failure
+## Notify datadog of a deploy failure
 datadog\:notify-deploy-failure:
-	$(call datadog_notify,$(subst datadog:notify-,,$@))
+	$(call datadog_notify_deploy,$(subst datadog:notify-,,$@))
+
+define datadog_notify_release
```

Comment:
I would think these release notifications are specific to the release-harness, why not have them there?